gatsby-config.js

```diff
@@ -56,5 +57,61 @@ module.exports = {
         path: `${__dirname}/src/data`,
       },
     },
+    {
+      resolve: 'gatsby-plugin-feed',
+      options: {
+        query: `
+          {
+            site {
+              siteMetadata {
+                title
+                description
+                siteUrl
+                site_url: siteUrl
```

Comment:
why are we exposing `siteUrl` via both snake and camel case?

Reply:
@FWirtz This is following the example provided here: https://www.gatsbyjs.org/docs/adding-an-rss-feed/#customizing-the-rss-feed-plugin

`site_url` is used to generate the `<link>` tag for the channel. Without it being exposed, the value defaults to https://github.com/dylang/node-rss. It seems that exposing `siteUrl` is not necessary if you use `site_url` in the custom `serialize` function.

So if you'd like it could be consolidated around `site_url`. It appears `node-rss` uses quite a bit of snake case whereas the Gatsby plugin tries to only use camel case and that conflict of style is what results in both being used in the example.

Reply:
phew that's annoying. Thanks for explaining!
I think it's alright to leave it.
Not sure the getting rid of the second variable is worth the using snake case instead.